### ISSUE TITLE
Why do we add a None when processing logits

### ISSUE THREAD

**Issue by zkokaja** [hassonlab/247-pickling#125]
https://github.com/hassonlab/247-pickling/blob/b8253c6a5088c5b2f8e75e6a5cf7f3440b82cbf5/scripts/tfsemb_main.py#L154
